scripts/process/simplify_wu_data.R

```diff
@@ -7,24 +7,10 @@ process.simplify_wu_data <- function(viz) {
   orig_wu_type_col <- deps[["col_names"]][["orig_names"]]
   new_wu_type_col <- deps[["col_names"]][["new_names"]]
   
-  columns_to_keep <- c("STATE", "STATEFIPS", "COUNTY", "COUNTYFIPS", "FIPS", "YEAR") # also double as character cols
+  columns_to_keep <- c("STATE", "STATEFIPS", "COUNTY", "COUNTYFIPS", "FIPS", "YEAR")
   columns_to_rename <- c("TP-TotPop", orig_wu_type_col)
   new_names <- c("countypop", new_wu_type_col)
   
-  # none of the columns we kept had characters (-- or N/A) that caused the entire column to
```

Comment:
wait, I think you need to pull from upstream. All of this stuff got deleted.

Reply:
I think I reverted some change I made in that file...but that wiped out my upstream pull....so now I need to figure out how to un-revert